oauth2/src/main/java/com/okta/spring/boot/oauth/Okta.java

```diff
@@ -162,22 +212,31 @@ static String statusAsString(HttpStatus status) {
         return status.value() + " " + status.getReasonPhrase();
     }
 
-    @Value("${okta.oauth2.audience:}")
-    private static String audience;
-
-    private static ServerOAuth2AuthorizationRequestResolver authorizeRequestResolver(
-        ReactiveClientRegistrationRepository clientRegistrationRepository) {
+    private static ServerOAuth2AuthorizationRequestResolver reactiveAuthzRequestResolver(
+        ReactiveClientRegistrationRepository clientRegistrationRepository, String audience) {
 
         DefaultServerOAuth2AuthorizationRequestResolver authorizationRequestResolver =
```

Comment:
How do you get the `authorizationRequestResolver` if it already exists? If I could figure that out, I could probably solve the chaining problem where the audience parameters overwrite the PKCE parameters.